### PR TITLE
task1 boiler plate

### PR DESCRIPTION
**Pull request by FunkyColdMedyna**
Hi Team, 

I made an attempt to set up our boiler plate for the team react app. Please take a look and let me know your thoughts. I initially ran into some problems trying to save time utilizing the command line and "create-react-app"; but this lead to some errors, so i went back through and tried to set everything up manually.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (hjkmines) [inline package.json:1]: @FunkyColdMedyna nice work setting up the boiler plate! 

No need to push up the package-lock.json in the repo, let's add this in our .gitignore

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (hjkmines) [inline .gitignore:1]: @FunkyColdMedyna nice! Let's add another one here for: package-lock.json

**Review by hjkmines — CHANGES_REQUESTED** [review]

**Comment by FunkyColdMedyna** [comment]
added package-lock.json to .git_ignore

**Comment by hjkmines** [comment]
LGTM! @FunkyColdMedyna

**Review by hjkmines — APPROVED** [review]

**Review by jordanclrk74 — APPROVED** [review]

**Review by jordanclrk74 — APPROVED** [review]

**Review by mattstead6 — COMMENTED** [review]
all good!

**Comment by JosephThawn** [comment]
Looks good to me!

**Pull request merged**